.betterer.results

```diff
@@ -236,10 +236,10 @@ exports[`no enzyme tests`] = {
     "public/app/plugins/datasource/cloud-monitoring/components/Aggregation.test.tsx:3822406835": [
       [1, 19, 13, "RegExp match", "2409514259"]
     ],
-    "public/app/plugins/datasource/cloudwatch/components/ConfigEditor.test.tsx:1224072551": [
-      [0, 19, 13, "RegExp match", "2409514259"]
+    "public/app/plugins/datasource/cloudwatch/components/ConfigEditor.test.tsx:4293633185": [
+      [1, 19, 13, "RegExp match", "2409514259"]
```

Comment:
is that `1` a bad thing?
Does that mean we added an enzyme test? Could we have avoided it?